test_runner/regress/test_pageserver_restart.py

```diff
@@ -109,6 +115,11 @@ def test_pageserver_restart(neon_env_builder: NeonEnvBuilder):
 # safekeeper and compute node keep running.
 @pytest.mark.timeout(540)
 def test_pageserver_chaos(neon_env_builder: NeonEnvBuilder):
+    neon_env_builder.enable_remote_storage(
+        remote_storage_kind=s3_storage(), test_name="test_pageserver_restart"
```

Comment:
```suggestion
        remote_storage_kind=s3_storage(), test_name="test_pageserver_chaos"
```

Alternatively, we can use `sys._getframe().f_code.co_name` everywhere, which returns the name of the current function

Reply:
This manual test name typing annoyed me enough that I went and fixed it everywhere: https://github.com/neondatabase/neon/pull/5184